src/type/introspection.ts

```diff
@@ -323,6 +323,16 @@ export const __Type: GraphQLObjectType = new GraphQLObjectType({
         type: __Type,
         resolve: (type) => ('ofType' in type ? type.ofType : undefined),
       },
+      oneOf: {
+        type: GraphQLBoolean,
+        resolve: (type) => {
+          if (isInputObjectType(type)) {
+            return type.isOneOf;
```

Comment:
We should have matching names in introspection and in JS types.
So we either should have both be `oneOf` or both being `isOneOf`.
Personally, I like `isOneOf` + it matching the `@deprecated`/`isDeprecated` pair that we already have.

Reply:
Updated in spec and implementation to use `isOneOf` for consistency with `isDeprecated` :+1: